product_docs/docs/biganimal/release/getting_started/identity_provider/aws_sso.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Using AWS SSO as your identity provider"
+navTitle: "AWS SSO"
+description: "Using AWS SSO as your identity provider for your Azure account"
+---
+
+## Prerequisites
+
+- To connect BigAnimal to Amazon Web Services Signle Sign-On(AWS SSO), your AWS account must have administrator access. 
+
+- A unique URL and access code are provided in an email from cloudcare@enterprisedb.com. Contact [cloudcare@enterprisedb.com](mailto:cloudcare@enterprisedb.com) if you don't receive the email. The URL becomes invalid after you set up your identity provider with BigAnimal. For issues with the code or identity provider setup, contact [Support](/biganimal/release/overview/support).
+
+## Set up BigAnimal with AWS SSO
+
+1. To access the Set Up Identity Provider page in BigAnimal, open the link in the email sent from cloudcare@enterprisedb.com.
+1. You will be copying and pasting between BigAnimal and AWS so in a separate browser tab or window log into your AWS account and go to the [**AWS SSO console**](https://console.aws.amazon.com/singlesignon).
+1. Choose **Enable AWS SSO** or if SSO is already enabled on your account, then continue to the next step. 
+   !!! note
+   You may need to switch to a region where SSO is enabled. AWS Organizations support SSO in only one region at a time.
+   !!!
+1. From the left navigation pane, select **Applications** or select the link in Step 3 on the **Dashboard**. 
```

Comment:
Navigate to the Applications page by either selecting  **Applications** from the left navigation pane or selecting the link in step 3 on the Welcome to AWS Single Sign-On page.